pkg/cmd/utility/utility.go

```diff
@@ -22,8 +22,8 @@ func AddGitSuffixIfNecessary(url string) string {
 	if url == "" || strings.HasSuffix(strings.ToLower(url), ".git") {
 		return url
 	}
-	log.Italicf("Adding .git to %s", url)
-	return url + ".git"
+	notifyUsers(url)
```

Comment:
I think we can improve this,

Let's merge the behaviour of `notifyUsers` back into this, but do something like...

```Go
trimmed := strings.TrimSuffix(url, "/")

if trimmed != url {
  log.Italicf(`Trimmed "/" from the end of %q", url)
}

return trimmed + ".git"